web/packages/design/src/Popover/Popover.story.tsx

```diff
@@ -39,8 +39,19 @@ export const Tooltip = () => (
   </Box>
 );
 
-class SimplePopover extends React.Component {
-  state = {
+type SimplePopoverState = {
+  anchorEl: Element | null;
+  anchorOrigin?: Origin;
+  transformOrigin?: Origin;
+  growDirections?: GrowDirections;
+  contentMultiplier: number;
+};
+
+class SimplePopover extends React.Component<any, SimplePopoverState> {
+  btnRef: Element;
+  growContentTimer: ReturnType<typeof setInterval>;
```

Comment:
I don't see any other references to these, where are they used?

Reply:
@ryanclark These are used in `startGrowContent`, line 145, which is called in `bottomRightGrowDirection` and `topLeftGrowDirection`. These are functions in our story that show what the popover looks like when its content changes dimensions.

Reply:
(And `btnRef` is used throughout this class in various positioning methods like `bottomCenter` etc., also used in the "Sample" story.)

Reply:
Ahhh that explains it. Cheers